src/Vault/Vault.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageProjectUrl>https://github.com/Chatham/Vault.NET/</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/Chatham/Vault.NET/blob/master/LICENSE</PackageLicenseUrl>
     <PackageIconUrl>https://github.com/Chatham/Vault.NET/blob/master/vault.png?raw=true</PackageIconUrl>
-    <PackageVersion>0.9.1.0</PackageVersion>
+    <PackageVersion>0.9.2.0</PackageVersion>
```

Comment:
We mirror the Vault version and only create new point releases for PRs and such that are not lined up with a newer version of Vault. Could you move this to `0.9.1.1`?